src/main/java/frc/robot/NetworkTables/NetworkTablesBase.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.NetworkTables;
+
+import edu.wpi.first.networktables.EntryListenerFlags;
+//imports
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableEntry;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.networktables.TableEntryListener;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+//Class
+public class NetworkTablesBase {
+
+  NetworkTableInstance inst;
+  NetworkTableEntry xEntry;
+  NetworkTableEntry yEntry;
+  TableEntryListener Ylistener;
+  TableEntryListener Xlistener;
+  NetworkTable piTable;
+
+  public NetworkTablesBase() {
+    run();
+  }
+
+  public static void run() {
+    NetworkTableInstance inst = NetworkTableInstance.getDefault();
+    NetworkTable table = inst.getTable("datatable");
+    NetworkTableEntry xEntry = table.getEntry("x");
```

Comment:
Don't define variables of the same name in two different scopes. Also, is this variable even needed?